substrate/client/rpc-spec-v2/src/chain_head/chain_head.rs

```diff
@@ -357,25 +404,26 @@ where
 		let mut items = items;
 		items.truncate(num_operations);
 
-		let (rp, rp_is_success) = method_started_response(operation_id, Some(discarded));
-
+		let (rp, rp_fut) = method_started_response(operation_id, Some(discarded));
 		let fut = async move {
-			// Events should only by generated
-			// if the response was successfully propagated.
-			if rp_is_success.await.is_err() {
+			// This leaves on a seperate task, because the `rp` needs to be acknowledged by the
```

Comment:
Wait for the server to send out the response and if it produces an error no event should be generated

Reply:
Yep I do like that phrasing better, thanks!